UnityProject/Assets/Scripts/Items/Science/Implants/BreathingTubeImplant.cs

```diff
@@ -0,0 +1,25 @@
+using HealthV2;
+using Mirror;
+
+namespace Items.Implants.Organs
+{
+	public class BreathingTubeImplant : BodyPartFunctionality
```

Comment:
Is net behaviour?

Reply:
odyPartFunctionality is Netbehaviour yeah